typescript/types/main.d.ts

```diff
@@ -1584,7 +1584,7 @@ declare class NRF {
    * @param {any} callback - A callback function to be called when the data is sent
    * @url http://www.espruino.com/Reference#l_NRF_sendHIDReport
    */
-  static sendHIDReport(data: any, callback: any): void;
+  static sendHIDReport(data: number[], callback?: () => void): void
```

Comment:
Hope you don't mind - took the liberty of regenerating `main.d.ts` from my [PR in Espruino](https://github.com/espruino/Espruino/pull/2357)